src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -1593,14 +1573,18 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
         case OP_TYPEID::LogicalAnd_v1:
         case OP_TYPEID::LogicalOr_v1:
         case OP_TYPEID::LogicalXor_v1:
+        case OP_TYPEID::Max:
         case OP_TYPEID::Maximum:
+        case OP_TYPEID::Min:
         case OP_TYPEID::Minimum:
         case OP_TYPEID::Multiply:
         case OP_TYPEID::NonZero_v3:
         case OP_TYPEID::NotEqual:
         case OP_TYPEID::Or:
         case OP_TYPEID::Power:
         case OP_TYPEID::Range:
+        case OP_TYPEID::ReduceMax_v1:
```

Comment:
Don't need these when you take them out out opset_in_tbl.

Reply:
Done